ruby/example_code/dynamodb/GettingStarted/MoviesCreateTable.rb

```diff
@@ -0,0 +1,72 @@
+# snippet-sourcedescription:[MoviesCreateTable.rb demonstrates how to ]
```

Comment:
This exact example already exists: 

https://github.com/awsdocs/aws-doc-sdk-examples/blob/master/ruby/example_code/dynamodb/dynamodb_ruby_example_create_movies_table.rb

Reply:
My mistake. I missed the endpoint tweak. I'll add it to the Ruby section as dynamodb_ruby_example_create_movies_table2.rb, with the snippet tag dynamodb.Ruby.CodeExample.MoviesCreateTable.

Reply:
okay...I am closing this merge request.... 
I am utterly confused (newbie here) why 1) other folks are checking duplicate samples 2) why the files are being moved around like that 

I don't believe i understand the process. I won't make any pull requests until i understand the process here.